autoendpoint/src/routers/apns/error.rs

```diff
@@ -86,3 +88,31 @@ impl From<ApnsError> for ApiErrorKind {
         ApiErrorKind::Router(RouterError::Apns(e))
     }
 }
+
+impl ReportableError for ApnsError {
+    fn reportable_source(&self) -> Option<&(dyn ReportableError + 'static)> {
+        None
+    }
+
+    fn backtrace(&self) -> Option<&Backtrace> {
+        None
+    }
+
+    fn is_sentry_event(&self) -> bool {
+        matches!(
+            self,
+            ApnsError::SizeLimit(_) | ApnsError::Unregistered | ApnsError::ApnsUpstream(_)
```

Comment:
Yeah, looks like a merge error, but I'm going to include `ApnsUpstream` as a valid Sentry error. We shouldn't get many of them, and if we do, it would impact our delivery capacity for APNS, so we should at least record the error.